### ISSUE TITLE
Interferes with image picker on other engines

### ISSUE THREAD

**Issue by cwise** [botandrose/refinerycms-ckeditor#5]
I have a custom engine with an image field. Install ckeditor interferes with the image picker functionality. The drag and drop is nice for the page editor but not at the expense of image fields on other engines.

**Comment by botandrose** [comment]
@cwise Makes sense. Hmm, how about adding a configuration flag that disables the image picker?

**Comment by cwise** [comment]
You mean the custom image picker from here? Yes that would make sense. The standard Refinery image picker is fine.

**Comment by botandrose** [comment]
Exactly. Feel like whipping up a pull request?

**Comment by cwise** [comment]
Sure. I'll work on it.
